src/features/createSurvey/addSurvey/AddSurveyForm.tsx

```diff
@@ -0,0 +1,83 @@
+import { type ChangeEvent, useState } from 'react';
+import { css, type Theme } from '@emotion/react';
+
+import ChoiceForm from '~/features/createSurvey/addSurvey/choiceForm/ChoiceForm';
+import TextToggle from '~/features/createSurvey/addSurvey/TextToggle';
+import { HEAD_1, HEAD_2_BOLD } from '~/styles/typo';
+
+const TOGGLE_LIST = [
+  {
+    type: 'CHOICE',
+    label: '객관식',
+  },
+  {
+    type: 'SHORT_ANSWER',
+    label: '주관식',
+  },
+];
+
+const AddSurveyForm = () => {
+  const [selectToggleTab, setSelectToggleTab] = useState(TOGGLE_LIST[0].type);
```

Comment:
```tsx
  const [selectToggleTab, setSelectToggleTab] = useState(TOGGLE_LIST[0].type);
```

이 상태에서는 타입 추론이 어떻게 되나요 ??

---

```suggestion
const TOGGLE_LIST = [
  {
    type: 'CHOICE',
    label: '객관식',
  },
  {
    type: 'SHORT_ANSWER',
    label: '주관식',
  },
] as const;

const AddSurveyForm = () => {
  const [selectToggleTab, setSelectToggleTab] = useState(TOGGLE_LIST[0].type);
```

이렇게 리드 온리로 선언하면 어떨까요 ~

Reply:
![스크린샷 2023-05-25 오후 2 59 39](https://github.com/depromeet/na-lab-client/assets/49177223/f21761ec-4096-4c5d-83ef-e750b3b16d8b)
위처럼 오류가 발생하는데, 해결 방안을 찾지 못했습니다 ㅜ